website/cue/reference/releases/0.25.0.cue

```diff
@@ -9,11 +9,11 @@ releases: "0.25.0": {
 	known_issues: [
 		"""
 			Vector fails to load configurations using environment variables outside of string values
-			for configuration options. This will be fixed in 0.25.1.
+			for configuration options. Fixed in 0.25.1.
 			""",
 		"""
 			Vector fails to load multi-file configurations using the global `timezone` configuration
-			option. This will be fixed in 0.25.1.
+			option. Fixed in 0.25.1.
 			""",
 	]
```

Comment:
:thinking: does this need to merge in https://github.com/vectordotdev/vector/commit/a62e7d64954321c67bab546dc21b9e3abc4be069 ?

Reply:
I was actually just doing this but forgot to mention it in the TODOs. Good spot.